components/service/Services.tsx

```diff
@@ -6,12 +8,21 @@ import { Service } from '@/types';
 const Services = async () => {
   const services = await getServices();
 
+  if (!services) {
+    notFound();
+  }
+
   return (
     <section className="flex flex-col items-center justify-center bg-white-800 px-[5.3125rem] py-[4.5rem] dark:bg-black-300">
       <ServiceHeading />
       <div className="flex flex-wrap  justify-center gap-x-4 gap-y-[2.5rem] md:justify-around ">
         {services?.map((card: Service) => (
-          <ServiceCard key={card?._id} {...card} />
+          <ServiceCard
+            key={card?._id}
+            imgSrcWhite={card?.whiteImage}
+            imgSrcBlue={card?.blueImage}
+            {...card}
+          />
```

Comment:
A few things to consider here... While optional chaining is great, you might also consider using nullish coalescing. This will provide a default value in case any of the properties are 'null' or 'undefined.' (good practice)